scripts/logrotate.config

```diff
@@ -0,0 +1,11 @@
+/var/log/nginx/*.log {
+        size 400M
+        rotate 10
+        notifempty
+        missingok
+        create 640 nginx nginx
+        compress
+        postrotate
```

Comment:
In umc-core feed we used to have only only access logs, so the USR1 directive would only ever be invoked once on access.log rotation.
Given that this is meant to be generic, we may have multiple type of logs, so `sharedscrip` would be best